### DISCUSSION TITLE
[Protokoll] 19.03.2024

### DISCUSSION THREAD

**Discussion by 7ben18** [AdversarialAttacks/.github#19] [Protokolle]
### Wer hat an dieser Sitzung teilgenommen?

- [X] Gabriel Torres Gamez
- [X] Si Ben Tran
- [X] Daniel Perruchoud
- [X] Stephan Heule

### Welche sind die wichtigsten Themen, die an dieser Sitzung besprochen wurden?

- Modellselektion, warum wir die Bildklassifikationsfamilien Modelle genommen haben. (Aufgrund unterschiedlicher Architektur und Aufbau) 
- Besprechung der Projektvereinbarung und fein Justierungen in Kapitel: 
-- 5. Lieferobjekte (Literaturrecherche, Einleitung in das Thema)
-- 4.1 Modelle (Ergänzung von ViT + EfficientNetV2)
-- 10 Risiken (Ergänzung von Massnahmen bei eintreten des Risiko)

### Was wurde an der Sitzung beschlossen?

- Projektvereinbarung finalisieren und ihnen als pdf senden, damit Dani und Stephan digital unterschreiben können. 
- Unterschriebene Projektvereinbarung wird im Teams unter "Dateien/02-Projektvereinbarungen" hinterlegt. 
- Nächste voraussichtliche Sitzung mit Dani und Stephan am 09.04.2024.
- Bis dahin Stephan informieren, ob wir die Sitzung (26.04.2024) benötigen, Spontanität von seiner Seite aus ist gegeben. 
- Modelle auf unseren Datensatz trainieren

**Comment by gabrieltorresgamez** [comment]
## Risiken 

- [x] Risiken: Bewusst, dass das Risiko vorhanden ist, jedoch haben wir keine Massnahme definiert, falls diese eintreffen. 

Massnahmen wurden eingepflegt.

## Danis Bemerkung
- Lieferobjekte -> Gut gemacht
- Termine: Viel Sommerzeit
- Zweite Hälfte Juni: Meilenstein 8:
- Zweite Hälfte Juno : Struktur von Dokument ihnen geben. 
- Feedback von Ihnen erhalten, bevor wir voll durchstarten beim Bericht.  
- Unbedingt einplanen, dass die Betreuer Zugriff auf unser Report haben und einen - Zwischenstand haben und wir im Sommer Vollgas arbeiten können. 

## Literaturrecherce

- [x] Literaturrecherche explizit in der Projektvereinbarung -> Methodischer Teil, Letzter Satz in der Methode. 

Anpassung in einem passenderen Kapitel, 5. Lieferobjekte. 
![image](https://github.com/AdversarialAttacks/.github/assets/66916399/a6646e34-d739-4e1a-abc2-fd27dcb58a52)


##  Unterschreiben Projektvereinbarung
Dokument wird digital Unterschrieben. 

- [ ] Alle haben das Dokument unterschrieben

## Technologie und Ressourcen
Hardware: Eigene Hardware GPU von Gabu und Ben. 
Technologie und Ressourcen: Kurze Besprechung wegen Hardware, ist sonst okay. 

## Modell: Evaluationsprotokoll

- [x] Modelle kurz beschrieben: Grundsätzlich unterschiedliche Klassifikationsmodelle. 

Beschreibung gemacht:
![image](https://github.com/AdversarialAttacks/.github/assets/66916399/436089e5-2d67-4846-9701-062fd4ea0637)


## Nächste Schritte: 
Modell Trainieren -> Fine Tunen-> Danach angriff.
Potentiell auf Huggingface hochladen für Drittpersonen. 

## Fragestellung
Dani hat durchgelesen, tüngt im gut. 

## Stephan und Dani Bemerkung
- Messbarkeit ist stabiler und besser packbar. Wenn wir merken, dass wir nach 2 Monaten nicht vorwärts kommen, können wir noch das Projekt in eine richtige Richtung lenken. 
- Projektvereinbarung in Anhang der Bachelorarbeit (Kann man machen) 
- Grundgerüst muss stabil sein.

## Projektvereinabarung
Wird ihnen gesendet. 

## Nächste Sitzungstermine
- 09.04.2024 
- Potentiell Stephan anschreiben, wenn wir Hilfe benötigen. Spontanität ist ebenfalls möglich. 
- Osterferien von 01.04.2024 bis 07.04.2024
